frontends/web/src/routes/buy/info.tsx

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright 2018 Shift Devices AG
```

Comment:
If this is new, there's no copyright by "shift devices" and should be removed.

Reply:
> > TODO:
> > the exchagnes will go into the guide
> 
> @thisconnect will you do it here or in a separate PR?

Better if the guide is in a separate PR